'

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env node_modules/.bin/ts-node
```

Comment:
1. all other files are not using this shebang, I think it's better to keep it consistent
2. I think you can find a better name for this file 😂